win32ss/gdi/ntgdi/dibobj.c

```diff
@@ -582,7 +585,7 @@ NtGdiSetDIBitsToDeviceInternal(
 
     SourceSize.cx = bmi->bmiHeader.biWidth;
     SourceSize.cy = ScanLines;
-    if (YDest >= 0 && YSrc > 0)
+    if (YDest >= 0 && YSrc >= 0)
     {
         ScanLines += YSrc;
     }
```

Comment:
How come adding `0` help?

Reply:
Since we are now setting "YSrc = 0;" sometimes in our new code, this affect cancels out our gains in PR #5227 unless we change the ">" to ">=" here. Previously YSrc never seemed to be zero here.

Reply:
@SergeGautherie, I realize what you were pointing out here now. If "YSrc == 0" then we are adding zero to ScanLines which has no effect, so this does nothing for us here. I have now removed this changed line. Thanks.